test_files/addon_tests.py

```diff
@@ -1513,6 +1519,161 @@ def model_spawner(self):
 		# Test collection/group added
 		# Test loading from file.
 
+	def geonode_effect_spawner(self):
+		"""Test the geo node variant of effect spawning works."""
+		if bpy.app.version < (3, 0):
```

Comment:
I think we need a `maxBv` function similar to the `minBv` function added alongside the optimizer. It has a benefit of also being cacheable, something that would help a lot

Reply:
Good catch to ask about this function. I don't think a maxBv is needed, as we can take any min bv and just inver the logic. You can see the change I made in [this commit](https://github.com/TheDuckCow/MCprep/commit/0a134a68636663db2b0ef66c46600d3601eab35f).